assets/scss/_footer.scss

```diff
@@ -3,17 +3,83 @@
 
 #footer {
   z-index:100;
-  padding:5vw;
-  background: $light_yellow;
+  .homepage {
+    &:before {
```

Comment:
Please remove this :before code - the `width: 200vw; left: -100vw` centering trick introduces horizontal scroll bars because the content is larger than the window. See notes above on how to achieve centering and preserve the background color.